connect/eaas/dataclasses.py

```diff
@@ -55,31 +55,48 @@ class TaskPayload:
     correlation_id: str = None
     reply_to: str = None
 
-    def to_json(self):
+    def to_dict(self):
         return dataclasses.asdict(self)
 
+    @classmethod
+    def from_dict(cls, data):
```

Comment:
Honestly for me dataclasses are something about the type and structure. That's why I don't think it is needed for some methods here. Especially if you need to copy it across all dataclasses. Why not to use existing `dataclasses.asdict()` and implement one like `fromdict(cls, data)`?

Reply:
Done!